packages/office-ui-fabric-react/src/components/FocusTrapZone/examples/FocusTrapZone.Box.Example.tsx

```diff
@@ -18,50 +19,42 @@ export class FocusTrapZoneBoxExample extends React.Component<{}, IFocusTrapZoneB
   private _toggle = React.createRef<IToggle>();
 
   public render() {
+    const { useTrapZone } = this.state;
+
     return (
       <div>
-        {this.state.useTrapZone ? (
-          <FocusTrapZone>{this._internalContents()}</FocusTrapZone>
-        ) : (
-          // prettier-ignore
-          this._internalContents()
-        )}
+        <DefaultButton secondaryText="Focuses inside the FocusTrapZone" onClick={this._onButtonClickHandler} text="Sample Initiator" />
```

Comment:
I removed the buttons in my recent PR to avoid having multiple UI elements which do the same thing--though in this case I can see how the toggle doesn't do *exactly* the same thing since it's inside the zone and the button isn't. Maybe in this case, it would be better to replace the toggle with a button that just disables the trap zone? (And the different border still makes it possible to see the enabled/disabled state even without the toggle.)

Also, I'm not sure about the text "Sample Initiator"--I'd prefer something more action-oriented like "Trap focus" (or remove the secondary text and just say "Focus inside FocusTrapZone")

Reply:
I also want to be clear this is demonstrating focus returning to initiator. How about this:

![image](https://user-images.githubusercontent.com/26070760/56612572-91c5c500-65c9-11e9-8180-0cd9007e00cb.png)

Reply:
Maybe "Focus trap initiator"? Also might be good to have secondary text and/or separate explanatory text noting that if you enable the focus trap zone with this button (rather than the toggle), focus will return to the button when the zone is disabled.